src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -247,6 +247,9 @@ private async Task<ScanResult> Dispatch(IScanArguments arguments, CancellationTo
                     catch (TimeoutException timeoutException)
                     {
                         Logger.LogError(timeoutException.Message);
+
+                        // TODO: remove the below message after go cli detector improvement has been stabilized.  
+                        Logger.LogWarning("If your repository contains Go packages, your build time may be improved by activating fallback strategy: https://aka.ms/go-detection-fallback-strategy");
```

Comment:
A similar message should also be added to the Go Detector. They might not have timed out, but if their pipeline is taking a lot longer than before, they should still be able to see a log about it. We should print `your build time may be improved by activating fallback strategy: https://aka.ms/go-detection-fallback-strategy` whenever Go CLI was found in system within the detector itself.

Reply:
added the suggested log.